content/english/python-tensorflow/activity-1.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Activity 1 - Importing the Tensorflow library and datasets"
+date: 2020-09-09T16:19:17-07:00
+weight: 2
+draft: false
+---
```

Comment:
Please add prereq: <the name of the activity or Python basics concept required> 
for all pages (see other examples in UCSD_Interns branch).